src/components/Collections/MyCollections.tsx

```diff
@@ -55,7 +59,20 @@ export function MyCollections({ children, onSelect }: MyCollectionsProps) {
 
   const Collections = (
     <Skeleton visible={isLoading} animate>
-      {filteredCollections.map((c) => (
+      {ownedFilteredCollections.length > 0 && <Text weight="bold">Owned</Text>}
+      {ownedFilteredCollections.map((c) => (
+        <>
+          <NavLink
+            key={c.id}
+            className={classes.navItem}
+            onClick={() => selectCollection(c.id)}
+            active={collectionId === c.id}
+            label={<Text>{c.name}</Text>}
+          ></NavLink>
+        </>
+      ))}
```

Comment:
Smol change, otherwise the `key` prop won't work here
```suggestion
      {ownedFilteredCollections.map((c) => (
        <NavLink
          key={c.id}
          className={classes.navItem}
          onClick={() => selectCollection(c.id)}
          active={collectionId === c.id}
          label={c.name}
        />
      ))}
```